packages/core/chrome/core-chrome-browser-internal/src/ui/header/screen_reader_a11y.tsx

```diff
@@ -38,5 +38,5 @@ export const ScreenReaderRouteAnnouncements: FC<{
     }
   }, [breadcrumbs]);
 
-  return <EuiScreenReaderLive focusRegionOnTextChange>{routeTitle}</EuiScreenReaderLive>;
+  return <EuiScreenReaderLive aria-live="assertive">{routeTitle}</EuiScreenReaderLive>;
```

Comment:
@1Copenut I set `aria-live` to `assertive` to make up for the removed focus behavior - not sure if this was the right move or is overly aggressive. Do you mind testing? 🙇

Reply:
from my testing: it works maybe 75% of the time on Safari+VO but not 100% of the time (for whatever reason, no idea why). Would feel better if we had JAWS and NVDA data.